src/library/Uno.Themes.WinUI.Markup/Theme/CalendarView.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using Windows.UI;
+using Microsoft.UI.Text;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup;
+using Uno.Extensions.Markup.Internals;
+
+using ResourceKeyDefinitionAttribute = Uno.Themes.WinUI.Markup.ResourceKeyDefinitionAttribute;
+
+namespace Uno.Themes.Markup
```

Comment:
Can we generate simplified namespaces eg
namespace Uno.Themes.Markup;
This will reduce indentation

Reply:
Agreed! In fact, there is already an option for this in the generator: https://github.com/unoplatform/Uno.Themes/blob/dev/xygu/20231012/markup-generator/src/library/Uno.Themes.WinUI.Markup.Generator/Xaml/Generators/LightWeightSourceGen.cs#L677
I chose to not do it to maintain consistency with legacy code (for text-diff-ing), but we are past that phase already.
I will enable this option in the upcoming changes.